src/components/ElementSchemes/DataElementIdScheme.js

```diff
@@ -1,32 +1,40 @@
-import React, { useEffect, useState } from 'react'
-import { useConfig } from '@dhis2/app-runtime'
+import React from 'react'
+import { useDataQuery } from '@dhis2/app-runtime'
 import PropTypes from 'prop-types'
 import i18n from '@dhis2/d2-i18n'
 
-import { fetchAttributes } from '../../utils/helper'
 import { dataElementIdSchemeOptions } from '../../utils/options'
 import { SelectField } from '../'
 
+const schemeQuery = {
+    data: {
```

Comment:
I think `data` is not a good identifier, it could mean anything. I'd prefer to just call it `dataElements` (Don't see a need for abbreviating identifiers here)